src/services/activityReports.js

```diff
@@ -1229,6 +1145,18 @@ async function getDownloadableActivityReports(where, separate = true) {
           {
             model: Grant,
```

Comment:
Does this need all the columns from the Grants table?

Reply:
Not sure tbh but to play it safe for now probably.